### PR TITLE
Add a DIG item_action

### LINKED ISSUE #17788: Debug message when using entrenching tools
Press [a] to use entrenching tools, then the debug message jumps out.

```
DEBUG: Couldn't find item action named DIG

FUNCTION: get_action
FILE: src\item_action.cpp
LINE: 178
```

### PR DESCRIPTION
**Pull request by sethsimon**
Fixes #17788.

### CODE REVIEW

**Comment by mugling** [comment]
May also fix #17210?

File: src/iuse_actor.h [hunk 1]
```diff
@@ -865,4 +865,22 @@ class place_trap_actor : public iuse_actor
         iuse_actor *clone() const override;
 };
 
+class clear_rubble_actor : public iuse_actor
```
Comment (mugling) [inline src/iuse_actor.h]: Why does this need to be an `iuse_actor`? No extra data is loaded from JSON?

**Comment by mugling** [comment]
If the qualities code cannot correctly handle `iuse` functions this should perhaps be fixed as opposed to converting everything to an `iuse_actor`.

**Comment by sethsimon** [comment]
@mugling It doesn't have to be an `iuse_actor`, but it if isn't, then in the item action menu (when activating an entrenching tool), "Clear rubble" is never greyed out.

**Comment by mugling** [comment]
It shouldn't be an `iuse_actor` unless the invocation depends upon other values loaded from JSON

**Comment by sethsimon** [comment]
Okay, I'll change it to a regular iuse function.

**Comment by Coolthulhu** [comment]
It would be a good idea to drop the move cost for now - non-long actions shouldn't cost more than 500 moves.

**Comment by sethsimon** [comment]
Okay, I'll move it down to 100.

**Pull request merged**